info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-21T18:30:55.477Z"}
info: Connected to the database. {"timestamp":"2022-09-21T18:30:55.572Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-21T18:31:11.669Z"}
info: Connected to the database. {"timestamp":"2022-09-21T18:31:11.729Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-21T18:31:29.727Z"}
info: Connected to the database. {"timestamp":"2022-09-21T18:31:29.763Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-21T18:31:58.364Z"}
info: Connected to the database. {"timestamp":"2022-09-21T18:31:58.407Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at E:\Fundoo_test\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (E:\Fundoo_test\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (E:\Fundoo_test\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (E:\Fundoo_test\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (E:\Fundoo_test\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (E:\Fundoo_test\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (E:\Fundoo_test\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (E:\Fundoo_test\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\Fundoo_test\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at E:\Fundoo_test\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at E:\Fundoo_test\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (E:\Fundoo_test\src\middlewares\/auth.middleware.js:12:22)
    at Layer.handle [as handle_request] (E:\Fundoo_test\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\Fundoo_test\node_modules\express\lib\router\route.js:144:13)
    at newNoteValidator (E:\Fundoo_test\src\validators\/user.validator.js:29:5)
    at Layer.handle [as handle_request] (E:\Fundoo_test\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\Fundoo_test\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\Fundoo_test\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\Fundoo_test\node_modules\express\lib\router\layer.js:95:5)
    at E:\Fundoo_test\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (E:\Fundoo_test\node_modules\express\lib\router\index.js:346:12)
    at next (E:\Fundoo_test\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\Fundoo_test\node_modules\express\lib\router\index.js:175:3)
    at router (E:\Fundoo_test\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\Fundoo_test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\Fundoo_test\node_modules\express\lib\router\index.js:328:13)
    at E:\Fundoo_test\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\Fundoo_test\node_modules\express\lib\router\index.js:346:12)
    at next (E:\Fundoo_test\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\Fundoo_test\node_modules\express\lib\router\index.js:175:3)
    at router (E:\Fundoo_test\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\Fundoo_test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\Fundoo_test\node_modules\express\lib\router\index.js:328:13)
    at E:\Fundoo_test\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\Fundoo_test\node_modules\express\lib\router\index.js:346:12)
    at next (E:\Fundoo_test\node_modules\express\lib\router\index.js:280:10)
    at logger (E:\Fundoo_test\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (E:\Fundoo_test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\Fundoo_test\node_modules\express\lib\router\index.js:328:13)
    at E:\Fundoo_test\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\Fundoo_test\node_modules\express\lib\router\index.js:346:12)
    at next (E:\Fundoo_test\node_modules\express\lib\router\index.js:280:10)
    at E:\Fundoo_test\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (E:\Fundoo_test\node_modules\raw-body\index.js:231:16)
    at done (E:\Fundoo_test\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (E:\Fundoo_test\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:525:35)
    at IncomingMessage.emit (node:domain:489:12) 
    url - /api/v1/note/note 
    method - POST 
    IP - ::1
   {"timestamp":"2022-09-21T18:35:30.970Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at E:\Fundoo_test\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (E:\Fundoo_test\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (E:\Fundoo_test\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (E:\Fundoo_test\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (E:\Fundoo_test\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (E:\Fundoo_test\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (E:\Fundoo_test\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (E:\Fundoo_test\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\Fundoo_test\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at E:\Fundoo_test\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at E:\Fundoo_test\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at userAuth (E:\Fundoo_test\src\middlewares\/auth.middleware.js:12:22)
    at Layer.handle [as handle_request] (E:\Fundoo_test\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\Fundoo_test\node_modules\express\lib\router\route.js:144:13)
    at newNoteValidator (E:\Fundoo_test\src\validators\/user.validator.js:29:5)
    at Layer.handle [as handle_request] (E:\Fundoo_test\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\Fundoo_test\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\Fundoo_test\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\Fundoo_test\node_modules\express\lib\router\layer.js:95:5)
    at E:\Fundoo_test\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (E:\Fundoo_test\node_modules\express\lib\router\index.js:346:12)
    at next (E:\Fundoo_test\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\Fundoo_test\node_modules\express\lib\router\index.js:175:3)
    at router (E:\Fundoo_test\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\Fundoo_test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\Fundoo_test\node_modules\express\lib\router\index.js:328:13)
    at E:\Fundoo_test\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\Fundoo_test\node_modules\express\lib\router\index.js:346:12)
    at next (E:\Fundoo_test\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\Fundoo_test\node_modules\express\lib\router\index.js:175:3)
    at router (E:\Fundoo_test\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\Fundoo_test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\Fundoo_test\node_modules\express\lib\router\index.js:328:13)
    at E:\Fundoo_test\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\Fundoo_test\node_modules\express\lib\router\index.js:346:12)
    at next (E:\Fundoo_test\node_modules\express\lib\router\index.js:280:10)
    at logger (E:\Fundoo_test\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (E:\Fundoo_test\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\Fundoo_test\node_modules\express\lib\router\index.js:328:13)
    at E:\Fundoo_test\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\Fundoo_test\node_modules\express\lib\router\index.js:346:12)
    at next (E:\Fundoo_test\node_modules\express\lib\router\index.js:280:10)
    at E:\Fundoo_test\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (E:\Fundoo_test\node_modules\raw-body\index.js:231:16)
    at done (E:\Fundoo_test\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (E:\Fundoo_test\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:525:35)
    at IncomingMessage.emit (node:domain:489:12) 
    url - /api/v1/note/note 
    method - POST 
    IP - ::1
   {"timestamp":"2022-09-21T18:35:40.241Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-21T18:40:36.218Z"}
info: Connected to the database. {"timestamp":"2022-09-21T18:40:36.406Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-21T18:42:29.039Z"}
info: Connected to the database. {"timestamp":"2022-09-21T18:42:29.116Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-21T18:42:46.744Z"}
info: Connected to the database. {"timestamp":"2022-09-21T18:42:47.062Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-21T18:43:18.514Z"}
info: Connected to the database. {"timestamp":"2022-09-21T18:43:18.601Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-09-21T18:43:25.474Z"}
info: Connected to the database. {"timestamp":"2022-09-21T18:43:25.552Z"}
